src/sentry/tasks/relay.py

```diff
@@ -99,79 +97,146 @@ def update_config_cache(
         )
 
 
-def schedule_update_config_cache(
-    generate, project_id=None, organization_id=None, public_key=None, update_reason=None
-):
-    """
-    Schedule the `update_config_cache` with debouncing applied.
+@instrumented_task(
+    name="sentry.tasks.relay.build_config_cache",
+    queue="relay_config",
+    acks_late=True,
+    soft_time_limit=30,
+    time_limit=32,
+)
+def build_config_cache(public_key=None, trigger=None, **kwargs):
```

Comment:
The invalidation task is now called `invalidate_project_config` so maybe this should be `build_project_config` for consistency?